### DISCUSSION TITLE
Enable search in public nested documents

### DISCUSSION THREAD

**Discussion by tvarsis** [outline/outline#2243] [General] [answered]
If I share a document collection, the search bar is not visible. The new feature of being able to share large and nested documents creates a strong need for the end users to be able to search through that public collection of documents, and the current limitation to search is strongly cripple the use of large public nested documents.

Expected result here is that we always show the same search bar, but that it is limited to the selected scope so that it only search on what the user have access to, and for public documents it would only search on the public document and their public nested documents.

**Comment by tommoor** [comment, marked as answer]
Totally agreed that there should be some search functionality exposed in shared nested documents. I do think the UI will need considering from the ground up though – likely the results good should in a flyover ontop of the page rather than be shown on an entirely separate screen.
